blurb/blurb.py

```diff
@@ -519,7 +519,7 @@ def _parse_next_filename(filename):
 Parses a "next" filename into its equivalent blurb metadata.
 Returns a dict.
         """
-        components = filename.split("/")
+        components = filename.split(os.sep)
```

Comment:
Should we also `os.path.normpath`?

Reply:
They're all obtained via glob, so it isn't going to be necessary.